e3sm_diags/e3sm_diags_driver.py

```diff
@@ -300,7 +301,7 @@ def _run_with_dask(parameters: List[CoreParameter]) -> List[CoreParameter]:
     https://docs.dask.org/en/stable/generated/dask.bag.map.html
     https://docs.dask.org/en/stable/generated/dask.dataframe.DataFrame.compute.html
     """
-    bag = dask.bag.from_sequence(parameters)
+    bag = db.from_sequence(parameters)
```

Comment:
For some reason this modules throws `AttributeError: module 'dask' has no attribute 'bag'` even though the dask version is the same as main (`dask=2023.12.1`). 

I don't know the root cause, but `import dask.bag as db` fixes the issue.